docs/Talkbuchet.js

```diff
@@ -0,0 +1,609 @@
+/**
+ *
+ * @copyright Copyright (c) 2021, Daniel Calviño Sánchez (danxuliu@gmail.com)
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+/**
+ * HOW TO SETUP:
+ * -----------------------------------------------------------------------------
+ * - Set the right values in "talkOcsApiUrl", "user" and "password" (a user must
+ *   be used; guests do not work).
+ * - If HPB clustering is enabled, set the token of a conversation in "token"
+ *   (otherwise leave empty).
+ * - Set whether to use audio, video or both in the "getUserMedia" call.
+ * - Set the desired numbers in "publishersCount" and
+ *   "subscribersPerPublisherCount" (in a regular call with N participants you
+ *   would have N publishers and N-1 subscribers).
+ *
+ * HOW TO RUN:
+ * -----------------------------------------------------------------------------
+ * - In the browser, log in the Nextcloud server (with the same user as in this
+ *   script).
+ * - Copy and paste the full script in the console of the browser to run it.
+ * - To run it again execute "closeConnections()" in the console; then you must
+ *   reload the page and copy and paste the script again.
+ *
+ * HOW TO ENABLE AND DISABLE THE MEDIA DURING A TEST:
+ * -----------------------------------------------------------------------------
+ * You can manually enable and disable the media during a test by copying and
```

Comment:
is this disabling for all or just the first one `[0]` ?

Reply:
There will be either zero or one stream of each type and the streams are shared between all publishers.